docs/cugraph/source/graph_support/algorithms/betweenness_centrality.md

```diff
@@ -0,0 +1,35 @@
+# Betweenness Centrality
+
+Betweenness centrality is a measure of the relative importance based on measuring the number of shortest paths that pass through each vertex or over each edge.  High betweenness centrality vertices have a greater number of path cross through the vertex.  Likewise, high centrality edges have more shortest paths that pass over the edge.
```

Comment:
you should introduce the short hand here:  "Between Centrality (BC) ..."

Reply:
Done